ip_repo/amdc_eddy_current_sensor_1.0/README.md

```diff
@@ -36,16 +38,18 @@ The IP is accessed via the AXI4-Lite register-based interface from the DSP. This
 | 31:18 | SIGN_EXT | Sign extended bit 17 |
 | 17:0  | DATA | 2's compliment 18 bit positional data of the Y direction |
 
-### CLOCK_DIVIDER
+### SPI_DIVIDER
 | Bits | Name | Description |
 | -- | -- | -- |
-| 7:0 | DIVIDER | Divides the SPI clock and conversion signal transition times by the value in the register plus one|
+| 7:0 | DIVIDER | SCLK will toggle after a number of AXI CLK cycles equal to this register value, default 10 <br /> Since AXI CLK period is 5ns, this will be 50ns high/50ns low, or 10MHz for SCLK |
 
-### CONVERSION_EN
+### PWM_TRIGGERS
 | Bits | Name | Description |
 | -- | -- | -- |
-| 0 | ENABLE | Enables the SPI clock and conversion signals to transition. A value of 0x1 enables the SPI interface. While enabled the IP core will continuously read from the eddy current sensor's ADCs |
-
-
-
+| 0 | PWM_HIGH_TRIGGER | If this bit is 1, a conversion/acquistion cycle will be triggered when the PWM carrier hits its max value (default 1) |
+| 1 | PWM_LOW_TRIGGER | If this bit is 1, a conversion/acquistion cycle will be triggered when the PWM carrier hits its min value (default 1) |
 
+### SHIFT_INDEX
+| Bits | Name | Description |
+| -- | -- | -- |
+| 7:0 | SHIFT_INDEX | Which flip-flop in the "shift" signal shift register we should use as our actual "shift" signal. <br /> This is needed to delay our shifting to align with the Kaman adapter board's propogation delay |
```

Comment:
Can you add a new section of this readme file which discusses the propagation delay, how to measure it, and how to convert the measured delay (one way?) to the SHIFT_INDEX value. This is bound to confuse people :)